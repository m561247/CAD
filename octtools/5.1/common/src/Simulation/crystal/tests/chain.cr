model slope
option printedgespeed
build chain.sim
in In
out Out
del In 0 -1
