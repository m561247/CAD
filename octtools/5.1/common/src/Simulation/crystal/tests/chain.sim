| units: 1	tech: nmos
e In A GND 400 400 100 100
d A A Vdd 1600 400 200 200
e A B GND 400 400 100 100
d B B Vdd 1600 400 200 200
e B C GND 400 400 100 100
d C C Vdd 1600 400 200 200
e C D GND 400 400 100 100
d D D Vdd 1600 400 200 200
e D E GND 400 400 100 100
d E E Vdd 1600 400 200 200
e E Out GND 400 400 100 100
d Out Out Vdd 1600 400 200 200
