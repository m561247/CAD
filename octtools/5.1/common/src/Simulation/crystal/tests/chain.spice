* Nodes 4-5 correspond to Out (see gate at 1,1)
* Nodes 6-7 correspond to E (see gate at 1,1)
* Nodes 8-9 correspond to D (see gate at 1,1)
* Nodes 10-11 correspond to C (see gate at 1,1)
* Nodes 12-13 correspond to B (see gate at 1,1)
* Nodes 14-15 correspond to A (see gate at 1,1)
* Nodes 16-17 correspond to In (see gate at 1,1)

C1 5 0 .00001pf
M1 1 5 5 2 d l=16.0u w=4.0u
M2 0 7 5 2 e l=4.0u w=4.0u
M3 0 9 7 2 e l=4.0u w=4.0u
M4 1 7 7 2 d l=16.0u w=4.0u
M5 1 9 9 2 d l=16.0u w=4.0u
M6 0 11 9 2 e l=4.0u w=4.0u
M7 0 13 11 2 e l=4.0u w=4.0u
M8 1 11 11 2 d l=16.0u w=4.0u
M9 1 13 13 2 d l=16.0u w=4.0u
M10 0 15 13 2 e l=4.0u w=4.0u
M11 0 17 15 2 e l=4.0u w=4.0u
M12 1 15 15 2 d l=16.0u w=4.0u

* Initial conditions:
.ic v(15)=5.000000
.ic v(13)=0.000000
.ic v(11)=5.000000
.ic v(9)=0.000000
.ic v(7)=5.000000
.ic v(17)=0.000000
.ic v(5)=0.000000

vin 17 0 pulse(0 5 0ns 0ns 0ns)
.tran 0.05ns 10ns
.plot tran V(5) (0,5)
.end
