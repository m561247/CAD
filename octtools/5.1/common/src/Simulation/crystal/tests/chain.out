1*******08/20/83 ********  SPICE 2G.6    3/15/83 ********20:35:06*****

0SPICE DECK GENERATED BY CRYSTAL.                                                

0****     INPUT LISTING                    TEMPERATURE =   27.000 DEG C

0***********************************************************************




 ** THE FOLLOWING LINES DEFINE THE NMOS MODELS AND VDD.  

 .MODEL E NMOS(VTO=.9 NSUB=2E15 TOX=.1U UO=800 XQC=.5    
  +LEVEL=2 VMAX=1E5 CGSO=245P CGDO=245P) 
 .MODEL D NMOS(VTO=-3.2 NSUB=2E15 TOX=.1U UO=800 XQC=.5  
  +LEVEL=2 VMAX=1E5 CGSO=245P CGDO=245P) 
 .OPTIONS NOMOD  
 .WIDTH OUT=80   
 VDD 1 0 5       
 VBB 2 0 0       
 * NODES 4-5 CORRESPOND TO OUT (SEE GATE AT 1,1) 
 * NODES 6-7 CORRESPOND TO E (SEE GATE AT 1,1)   
 * NODES 8-9 CORRESPOND TO D (SEE GATE AT 1,1)   
 * NODES 10-11 CORRESPOND TO C (SEE GATE AT 1,1) 
 * NODES 12-13 CORRESPOND TO B (SEE GATE AT 1,1) 
 * NODES 14-15 CORRESPOND TO A (SEE GATE AT 1,1) 
 * NODES 16-17 CORRESPOND TO IN (SEE GATE AT 1,1)
         
 C1 15 0 0.990PF 
 C2 13 0 0.040PF 
 C3 11 0 0.040PF 
 C4 9 0 0.190PF  
 C5 7 0 0.040PF  
 C6 17 0 0.042PF 
 C7 5 0 0.048PF  
 M1 1 5 5 2 D L=16.0U W=4.0U     
 M2 0 7 5 2 E L=4.0U W=4.0U      
 M3 0 9 7 2 E L=4.0U W=4.0U      
 M4 1 7 7 2 D L=16.0U W=4.0U     
 M5 1 9 9 2 D L=16.0U W=4.0U     
 M6 0 11 9 2 E L=4.0U W=4.0U     
 M7 0 13 11 2 E L=4.0U W=4.0U    
 M8 1 11 11 2 D L=16.0U W=4.0U   
 M9 1 13 13 2 D L=16.0U W=4.0U   
 M10 0 15 13 2 E L=4.0U W=4.0U   
 M11 0 17 15 2 E L=4.0U W=4.0U   
 M12 1 15 15 2 D L=16.0U W=4.0U  
         
 * INITIAL CONDITIONS:   
 .IC V(15)=5.000000      
 .IC V(13)=0.000000      
 .IC V(11)=5.000000      
 .IC V(9)=0.000000       
 .IC V(7)=5.000000       
 .IC V(17)=0.000000      
 .IC V(5)=0.000000       
         
 VIN 17 0 PULSE(0 5 0NS 0NS 0NS) 
 .TRAN 0.3NS 60NS
 .PLOT TRAN V(5) V(7) V(9) V(11) V(13) V(15) (0,5)       
 .END    
1*******08/20/83 ********  SPICE 2G.6    3/15/83 ********20:35:06*****

0SPICE DECK GENERATED BY CRYSTAL.                                                

0****     INITIAL TRANSIENT SOLUTION       TEMPERATURE =   27.000 DEG C

0***********************************************************************


  NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE


 (  1)    5.0000    (  2)    0.        (  5)    0.0000    (  7)    5.0000

 (  9)    0.0000    ( 11)    5.0000    ( 13)    0.0000    ( 15)    5.0000

 ( 17)    0.        




     VOLTAGE SOURCE CURRENTS

     NAME       CURRENT


     VDD      -8.443d-05

     VBB       8.320d-11

     VIN       0.   d+00


     TOTAL POWER DISSIPATION   4.22d-04  WATTS
1*******08/20/83 ********  SPICE 2G.6    3/15/83 ********20:35:06*****

0SPICE DECK GENERATED BY CRYSTAL.                                                

0****     OPERATING POINT INFORMATION      TEMPERATURE =   27.000 DEG C

0***********************************************************************


0
0**** MOSFETS


0            M1        M2        M3        M4        M5        M6        M7      
0MODEL     D         E         E         D         D         E         E       
 ID        2.81d-05 -3.19d-09  0.  d+00  1.93d-12  2.81d-05 -3.19d-09  0.  d+00
 VGS          0.        5.000    -5.000     0.        0.        5.000    -5.000
 VDS          5.000    -0.000    -5.000     0.000     5.000    -0.000    -5.000
 VBS         -0.000    -0.000    -5.000    -5.000    -0.000    -0.000    -5.000


0            M8        M9        M10       M11       M12     
0MODEL     D         D         E         E         D       
 ID        1.93d-12  2.81d-05 -3.19d-09  0.  d+00  1.93d-12
 VGS          0.        0.        5.000    -5.000     0.   
 VDS          0.000     5.000    -0.000    -5.000     0.000
 VBS         -5.000    -0.000    -0.000    -5.000    -5.000
1*******08/20/83 ********  SPICE 2G.6    3/15/83 ********20:35:06*****

0SPICE DECK GENERATED BY CRYSTAL.                                                

0****     TRANSIENT ANALYSIS               TEMPERATURE =   27.000 DEG C

0***********************************************************************


0LEGEND:

 *: V(5)    
 +: V(7)    
 =: V(9)    
 $: V(11)   
 0: V(13)   
 <: V(15)   
X
     TIME      V(5)    

(*+=$0<)--------  0.   d+00     1.250d+00     2.500d+00     3.750d+00  5.000d+00
                       - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 0.   d+00  2.814d-05 X             .             .             .             X
 3.000d-10  1.175d-01 =X            .             .             .             X
 6.000d-10  1.768d-01 .=X           .             .             .            <X
 9.000d-10  2.087d-01 .=X           .             .             .           < X
 1.200d-09  2.261d-01 .= X          .             .             .           < X
 1.500d-09  2.362d-01 . =X          .             .             .          <  X
 1.800d-09  2.414d-01 . =X          .             .             .         <   X
 2.100d-09  2.436d-01 . =X          .             .             .         <   X
 2.400d-09  2.459d-01 . =X          .             .             .        <    X
 2.700d-09  2.463d-01 . =X          .             .             .       <     X
 3.000d-09  2.465d-01 . =X          .             .             .       <     X
 3.300d-09  2.466d-01 . =X          .             .             .      <      X
 3.600d-09  2.468d-01 . =X          .             .             .     <       X
 3.900d-09  2.468d-01 . =X          .             .             .     <       X
 4.200d-09  2.468d-01 . =X          .             .             .    <        X
 4.500d-09  2.469d-01 .  X          .             .             .   <         X
 4.800d-09  2.469d-01 .  X          .             .             .   <         X
 5.100d-09  2.469d-01 .  X          .             .             .  <          X
 5.400d-09  2.469d-01 .  X0         .             .             .  <          X
 5.700d-09  2.470d-01 .  X0         .             .             . <           X
 6.000d-09  2.470d-01 .  X0         .             .             .<            X
 6.300d-09  2.470d-01 .  X0         .             .             .<            X
 6.600d-09  2.471d-01 .  X0         .             .             <             X
 6.900d-09  2.471d-01 .  X0         .             .            <.             X
 7.200d-09  2.471d-01 .  X0         .             .            <.             X
 7.500d-09  2.471d-01 .  X0         .             .           < .             X
 7.800d-09  2.472d-01 .  X0         .             .           < .             X
 8.100d-09  2.472d-01 .  X0         .             .          <  .             X
 8.400d-09  2.472d-01 .  X0         .             .          <  .             X
 8.700d-09  2.472d-01 .  X0         .             .         <   .             X
 9.000d-09  2.472d-01 .  X 0        .             .        <    .             X
 9.300d-09  2.473d-01 .  X 0        .             .        <    .             X
 9.600d-09  2.473d-01 .  X 0        .             .       <     .             X
 9.900d-09  2.473d-01 .  X 0        .             .       <     .             X
 1.020d-08  2.473d-01 .  X 0        .             .      <      .             X
 1.050d-08  2.473d-01 .  X 0        .             .      <      .             X
 1.080d-08  2.473d-01 .  X 0        .             .     <       .             X
 1.110d-08  2.473d-01 .  X 0        .             .    <        .             X
 1.140d-08  2.473d-01 .  X 0        .             .    <        .             X
 1.170d-08  2.473d-01 .  X  0       .             .   <         .             X
 1.200d-08  2.473d-01 .  X  0       .             .   <         .             X
 1.230d-08  2.473d-01 .  X  0       .             .  <          .             X
 1.260d-08  2.473d-01 .  X  0       .             .  <          .             X
 1.290d-08  2.474d-01 .  X  0       .             . <           .             X
 1.320d-08  2.474d-01 .  X   0      .             . <           .             X
 1.350d-08  2.474d-01 .  X   0      .             .<            .             X
 1.380d-08  2.474d-01 .  X   0      .             .<            .             X
 1.410d-08  2.474d-01 .  X   0      .             <             .             X
 1.440d-08  2.474d-01 .  X    0     .             <             .             X
 1.470d-08  2.474d-01 .  X    0     .            <.             .             X
 1.500d-08  2.474d-01 .  X    0     .            <.             .             X
 1.530d-08  2.474d-01 .  X    0     .           < .             .             X
 1.560d-08  2.474d-01 .  X     0    .           < .             .             X
 1.590d-08  2.474d-01 .  X     0    .          <  .             .             X
 1.620d-08  2.474d-01 .  X      0   .          <  .             .             X
 1.650d-08  2.474d-01 .  X      0   .         <   .             .             X
 1.680d-08  2.474d-01 .  X       0  .         <   .             .             X
 1.710d-08  2.474d-01 .  X       0  .        <    .             .             X
 1.740d-08  2.474d-01 .  X        0 .        <    .             .             X
 1.770d-08  2.474d-01 .  X         0.       <     .             .             X
 1.800d-08  2.474d-01 .  X         0.       <     .             .             X
 1.830d-08  2.474d-01 .  X          0       <     .             .             X
 1.860d-08  2.474d-01 .  X          .0     <      .             .             X
 1.890d-08  2.474d-01 .  X          . 0    <      .             .             X
 1.920d-08  2.474d-01 .  X          .  0  <       .             .            $+
 1.950d-08  2.474d-01 .  X          .  0  <       .             .            $+
 1.980d-08  2.474d-01 .  X          .   0<        .             .            $+
 2.010d-08  2.474d-01 .  X          .    X        .             .           $ +
 2.040d-08  2.474d-01 .  X          .    <0       .             .           $ +
 2.070d-08  2.474d-01 .  X          .   <  0      .             .          $  +
 2.100d-08  2.474d-01 .  X          .   <   0     .             .         $   +
 2.130d-08  2.474d-01 .  X          .   <    0    .             .         $   +
 2.160d-08  2.475d-01 .  X          .  <      0   .             .        $    +
 2.190d-08  2.475d-01 .  X          .  <       0  .             .       $     +
 2.220d-08  2.474d-01 .  X          . <         0 .             .     $       +
 2.250d-08  2.474d-01 .  X          . <          0.             .    $        +
 2.280d-08  2.474d-01 .  X          . <           0             .   $         +
 2.310d-08  2.474d-01 .  X          .<            .0            . $           +
 2.340d-08  2.474d-01 .  X          .<            . 0           $             +
 2.370d-08  2.474d-01 .  X          .<            .  0        $ .             +
 2.400d-08  2.474d-01 .  X          .<            .   0      $  .             +
 2.430d-08  2.474d-01 .  X          <             .    0  $     .             +
 2.460d-08  2.474d-01 .  *=         <             .     X       .             +
 2.490d-08  2.474d-01 .  *=         <             .  $   0      .             +
 2.520d-08  2.474d-01 .  *=        <.             $       0     .             +
 2.550d-08  2.474d-01 .  *=        <.          $  .        0    .             +
 2.580d-08  2.474d-01 .  *=        <.       $     .         0   .             +
 2.610d-08  2.474d-01 .  * =      < .     $       .          0  .             +
 2.640d-08  2.474d-01 .  * =      < .  $          .          0  .             +
 2.670d-08  2.474d-01 .  * =      < $             .           0 .             +
 2.700d-08  2.474d-01 .  *  =     X .             .            0.             +
 2.730d-08  2.473d-01 .  *  =   $<  .             .             0             +
 2.760d-08  2.473d-01 .  *  = $  <  .             .             .0            +
 2.790d-08  2.473d-01 .  *   X   <  .             .             .0            +
 2.820d-08  2.473d-01 .  *  $=   <  .             .             . 0           +
 2.850d-08  2.473d-01 .  * $  =  <  .             .             .  0          +
 2.880d-08  2.473d-01 .  *$   = <   .             .             .  0          +
 2.910d-08  2.472d-01 .  *$    =<   .             .             .   0         +
 2.940d-08  2.472d-01 .  *$    =<   .             .             .   0         +
 2.970d-08  2.472d-01 .  *$    =<   .             .             .    0        +
 3.000d-08  2.471d-01 .  *$    <=   .             .             .     0       +
 3.030d-08  2.471d-01 .  *$    <=   .             .             .     0       +
 3.060d-08  2.471d-01 .  *$    < =  .             .             .      0      +
 3.090d-08  2.471d-01 .  X     < =  .             .             .      0      +
 3.120d-08  2.471d-01 .  X     < =  .             .             .       0     +
 3.150d-08  2.471d-01 .  X     <  = .             .             .       0     +
 3.180d-08  2.471d-01 .  X    <   = .             .             .       0     +
 3.210d-08  2.471d-01 .  X    <    =.             .             .        0    +
 3.240d-08  2.471d-01 .  X    <    =.             .             .        0    +
 3.270d-08  2.471d-01 .  X    <    =.             .             .        0    +
 3.300d-08  2.472d-01 .  X    <     =             .             .         0   +
 3.330d-08  2.473d-01 .  X    <     =             .             .         0   +
 3.360d-08  2.478d-01 .  X   <      =             .             .         0   +
 3.390d-08  2.482d-01 .  X   <      .=            .             .          0  +
 3.420d-08  2.486d-01 .  X   <      .=            .             .          0 +.
 3.450d-08  2.491d-01 .  X   <      . =           .             .          0 +.
 3.480d-08  2.499d-01 .  X   <      . =           .             .          0 +.
 3.510d-08  2.506d-01 .  X   <      . =           .             .          0 +.
 3.540d-08  2.513d-01 .  X   <      .  =          .             .           0+.
 3.570d-08  2.522d-01 .  X   <      .  =          .             .           0+.
 3.600d-08  2.534d-01 .  X  <       .  =          .             .           X .
 3.630d-08  2.545d-01 .  X  <       .   =         .             .           X .
 3.660d-08  2.557d-01 .  X  <       .   =         .             .           X .
 3.690d-08  2.570d-01 .  X  <       .   =         .             .          +0 .
 3.720d-08  2.587d-01 .  X  <       .    =        .             .          +0 .
 3.750d-08  2.604d-01 .  X  <       .    =        .             .          + 0.
 3.780d-08  2.621d-01 .  X  <       .    =        .             .         +  0.
 3.810d-08  2.639d-01 .  X  <       .     =       .             .         +  0.
 3.840d-08  2.663d-01 .  X  <       .     =       .             .         +  0.
 3.870d-08  2.686d-01 .  X  <       .      =      .             .        +   0.
 3.900d-08  2.709d-01 .  X <        .      =      .             .        +   0.
 3.930d-08  2.734d-01 .  X <        .      =      .             .       +    0.
 3.960d-08  2.765d-01 .  X <        .       =     .             .       +    0.
 3.990d-08  2.796d-01 .  X <        .       =     .             .      +     0.
 4.020d-08  2.827d-01 .  X <        .       =     .             .      +     0.
 4.050d-08  2.862d-01 .  X <        .        =    .             .     +      0.
 4.080d-08  2.903d-01 .  X <        .        =    .             .     +      0.
 4.110d-08  2.944d-01 .  X <        .        =    .             .    +       0.
 4.140d-08  2.985d-01 .  X <        .         =   .             .    +       0.
 4.170d-08  3.031d-01 .  X <        .         =   .             .   +        0.
 4.200d-08  3.086d-01 .  X <        .         =   .             .  +          0
 4.230d-08  3.141d-01 .  $*<        .         =   .             .  +          0
 4.260d-08  3.196d-01 .  $*<        .          =  .             . +           0
 4.290d-08  3.257d-01 .  $*<        .          =  .             .+            0
 4.320d-08  3.332d-01 .  $*<        .          =  .             +             0
 4.350d-08  3.408d-01 .  $X         .           = .             +             0
 4.380d-08  3.483d-01 .  $X         .           = .            +.             0
 4.410d-08  3.568d-01 .  $X         .           = .           + .             0
 4.440d-08  3.674d-01 .  $X         .            =.          +  .             0
 4.470d-08  3.781d-01 .  $X         .            =.         +   .             0
 4.500d-08  3.888d-01 .  $X         .            =.        +    .             0
 4.530d-08  4.011d-01 .  $X         .             =       +     .             0
 4.560d-08  4.173d-01 .  $<*        .             =      +      .             0
 4.590d-08  4.335d-01 .  $<*        .             =     +       .             0
 4.620d-08  4.496d-01 .  $<*        .             =    +        .             0
 4.650d-08  4.690d-01 .  $<*        .             .=  +         .             0
 4.680d-08  4.956d-01 .  $< *       .             .= +          .             0
 4.710d-08  5.221d-01 .  $< *       .             .=+           .             0
 4.740d-08  5.487d-01 .  $< *       .             .+=           .             0
 4.770d-08  5.828d-01 .  $<  *      .            +. =           .             0
 4.800d-08  6.337d-01 .  $<  *      .           + . =           .             0
 4.830d-08  6.847d-01 .  $<   *     .          +  .  =          .             0
 4.860d-08  7.356d-01 .  $<   *     .         +   .  =          .             0
 4.890d-08  7.971d-01 .  $<    *    .       +     .  =          .             0
 4.920d-08  8.821d-01 .  $<     *   .      +      .  =          .             0
 4.950d-08  9.671d-01 .  $<      *  .     +       .   =         .             0
 4.980d-08  1.052d+00 .  $<       * .   +         .   =         .             0
 5.010d-08  1.145d+00 .  $<        *.  +          .   =         .             0
 5.040d-08  1.256d+00 .  $<         * +           .   =         .             0
 5.070d-08  1.367d+00 .  $<         .X            .    =        .             0
 5.100d-08  1.478d+00 .  $<        +.  *          .    =        .             0
 5.130d-08  1.591d+00 .  $<       + .   *         .    =        .             0
 5.160d-08  1.710d+00 .  X       +  .    *        .     =       .             0
 5.190d-08  1.828d+00 .  X      +   .     *       .     =       .             0
 5.220d-08  1.947d+00 .  X      +   .       *     .     =       .             0
 5.250d-08  2.063d+00 .  X     +    .        *    .     =       .             0
 5.280d-08  2.175d+00 .  X    +     .         *   .      =      .             0
 5.310d-08  2.286d+00 .  X    +     .           * .      =      .             0
 5.340d-08  2.398d+00 .  X   +      .            *.      =      .             0
 5.370d-08  2.507d+00 .  X   +      .             *      =      .             0
 5.400d-08  2.611d+00 .  X  +       .             .*      =     .             0
 5.430d-08  2.715d+00 .  X  +       .             . *     =     .             0
 5.460d-08  2.819d+00 .  X  +       .             .   *   =     .             0
 5.490d-08  2.920d+00 .  X  +       .             .    *  =     .             0
 5.520d-08  3.015d+00 .  X  +       .             .     *  =    .             0
 5.550d-08  3.111d+00 .  X  +       .             .      * =    .             0
 5.580d-08  3.207d+00 .  X +        .             .       *=    .             0
 5.610d-08  3.299d+00 .  X +        .             .        X    .             0
 5.640d-08  3.384d+00 .  X +        .             .         X   .             0
 5.670d-08  3.469d+00 .  X +        .             .         =*  .             0
 5.700d-08  3.554d+00 .  X +        .             .         = * .             0
 5.730d-08  3.635d+00 .  X +        .             .         =  *.             0
 5.760d-08  3.708d+00 .  X +        .             .          =  *             0
 5.790d-08  3.781d+00 .  X +        .             .          =  *             0
 5.820d-08  3.854d+00 .  X +        .             .          =  .*            0
 5.850d-08  3.923d+00 .  X +        .             .          =  . *           0
 5.880d-08  3.983d+00 .  X +        .             .           = .  *          0
 5.910d-08  4.044d+00 .  X +        .             .           = .  *          0
 5.940d-08  4.104d+00 .  X +        .             .           = .   *         0
 5.970d-08  4.163d+00 .  X +        .             .           = .    *        0
 6.000d-08  4.216d+00 .  X +        .             .            =.    *        0
                       - - - - - - - - - - - - - - - - - - - - - - - - - - - - -


Y
0
         JOB CONCLUDED
0         TOTAL JOB TIME           49.75
